plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/completion/SQLCompletionAnalyzer.java

```diff
@@ -664,8 +664,8 @@ private void makeProposalsFromQueryParts() {
             if (wordPart.indexOf(request.getContext().getSyntaxManager().getStructSeparator()) != -1 || wordPart.equals(ALL_COLUMNS_PATTERN)) {
                 return;
             }
-            final Pair<String, String> name = extractTableName(wordPart, true);
-            if (name != null) {
+            final List<Pair<String, String>> names = extractTableNames(wordPart, true);
+            for (Pair<String, String> name: names) {
```

Comment:
```suggestion
            for (Pair<String, String> name : extractTableNames(wordPart, true)) {
```

Reply:
I'd prefer to have this variable for debug. It used to be a separate variable and a don't like to change it, because it useful.